cli/src/main.rs

```diff
@@ -195,8 +198,14 @@ fn run() -> HolochainResult<()> {
             cli::test(&current_path, &dir, &testfile, skip_build)
         }
         .map_err(HolochainError::Default)?,
-        Cli::KeyGen => {
-            cli::keygen(None, None).map_err(|e| HolochainError::Default(format_err!("{}", e)))?
+        Cli::KeyGen { silent } => {
+            let passphrase = if silent {
```

Comment:
You can use the `filter` method on `Option<T>`:

```rust
let passphrase = Some(holochain_common::DEFAULT_PASSPHRASE.to_string()).filter(silent);
```

Reply:
Nope, that actually doesn't work.  filter takes a function not a boolean.  So I'm just going to merge as is.